src/main/AntidoteBackend.java

```diff
@@ -0,0 +1,114 @@
+package main;
+
+import com.ericsson.otp.erlang.OtpErlangBinary;
+import com.ericsson.otp.erlang.OtpErlangAtom;
+import com.ericsson.otp.erlang.OtpErlangPid;
+import com.ericsson.otp.erlang.OtpErlangTuple;
+import com.ericsson.otp.erlang.OtpMbox;
+import com.ericsson.otp.erlang.OtpNode;
+
+abstract class AntidoteBackend {
```

Comment:
implements Callable<void> ? Or Runnable, if that's still in use?

Reply:
Apparently Runnable is not deprecated? at least for the java I'm using